app/views/alaveteli_pro/info_requests/_info_request.html.erb

```diff
@@ -40,6 +40,11 @@
       <p><%= info_request.date_response_required_by.strftime('%d-%m-%Y') %></p>
     </div>
 
+    <div class="request__status">
```

Comment:
I think this needs a separate class as you've done for batches above because

* Its not aligned correctly on smaller screens
* It doesn't make sense semantically

![screen shot 2017-12-06 at 10 28 36](https://user-images.githubusercontent.com/282788/33657145-4093106a-da70-11e7-93ee-f6f4058dfe46.png)